Formula/protobuf@3.6.rb

```diff
@@ -15,6 +15,8 @@ class ProtobufAT36 < Formula
 
   keg_only :versioned_formula
 
+  deprecate! date: "2021-02-19", because: :unmaintained
```

Comment:
Another option is `because: :versioned_formula` which is maybe more descriptive 

https://github.com/Homebrew/brew/blob/master/docs/Deprecating-Disabling-and-Removing-Formulae.md#deprecate-and-disable-reasons

Reply:
Done.